### PR TITLE
Fix minor spelling issues and change restart pragma

### PR DESCRIPTION
**Pull request by frankhereford**
This is a small, vestigial PR from some prep work that I was doing on the postgrest upgrade. I think there is no testing required; just a code review.

The restart pragma change mirrors our pattern for the moped database. This allows us to stop the service if we want, but still have it restart if it becomes non-running for any other reason than one of us stopping it.

### CODE REVIEW

File: README.md
Comment (frankhereford) [inline README.md:1]: Save the two spelling changes in this file, the rest is linting.

File: docker-compose.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ services:
       - "./haproxy/maps:/etc/haproxy/maps:ro"
   postgrest-legacy-scripts:
     image: postgrest/postgrest
```
Comment (johnclary) [inline docker-compose.yaml:18]: this all looks great. do you think we should pin the postgrest versions, Frank? i am of two minds...

**Review by johnclary — APPROVED** [review]
thanks for cleaning this up and tweaking that restart setting!

File: docker-compose.yaml (hunk 1 above)
Comment (frankhereford) [reply docker-compose.yaml:18]: Yea, it's a great question and I don't have an opinion really. Until we intentionally `docker pull` on this, it'll stay the same so I'm inclined to leave it, but I would also be just as fine pinning it. Like you -- two minds.

File: README.md
Comment (chiaberry) [reply README.md:1]: happppppproxy

**Review by chiaberry — COMMENTED** [review]
thanks for the cleanup!

**Pull request merged**